### ISSUE TITLE
Errors during instantiation of request scoped services are swallowed (Nest Microservice)

### ISSUE THREAD

**Issue by AdScoops** [nestjs/nest#3720]
### Bug Report
If an injected request scoped service fails during instantiation a nest microservice using the client proxy will not know about these errors. This is an attempt at a very basic example of the issue I'm seeing using the client proxy to handle SQS messages in AWS Lambda. 

### Current Behavior
The crux of the issue is that unhandled exceptions during code execution will cause the lambda to fail allowing for the message to be reprocessed. However, if the failure occurs during the creation of a request scoped service then the error is swallowed and it doesn't seem like there is any way for the client proxy to know that the message will not be processed. The lambda then passes and the message is lost. 

### Reproduction
https://github.com/AdScoops/microservice-error
running as is will show a failure in the request scoped service but no exception caught in the try-catch in main.ts

### Expected Behavior
If an error is thrown during code execution it will trigger the catch block in main.ts. I would expect or like to be able to handle errors during creation of request scoped services in the same manner

Environment
```
@nestjs/common: "6.10.12"
@nestjs/microservices: "6.10.12"
Node: 10.13.0
```

**Comment by cojack** [comment]
Do not use `nest` cmd to start application, it swallow a lot of errors without any good reason, in script start replace it with ts-node with path to your main file.

**Comment by cojack** [comment]
Why my comment was deleted?

**Comment by kamilmysliwiec** [comment]
Because it was unrelated to this issue @cojack.

@AdScoops issue has been fixed in 6.10.14. Thanks for reporting.
